tests/app/views/contexts/test_calculated_summary_context.py

```diff
@@ -9,237 +7,115 @@
-        )
-
-        self.assertTrue("calculated_question" in context_summary)
-        self.assertEqual(
-            context_summary["calculated_question"]["title"],
-            "Grand total of previous values",
-        )
-        self.assertEqual(
-            context_summary["calculated_question"]["answers"][0]["value"], "20%"
-        )
-
-    @patch("app.jinja_filters.flask_babel.get_locale", Mock(return_value="cy"))
-    def test_build_view_context_for_number_calculated_summary(self):
-        current_location = Location(
-            section_id="default-section", block_id="number-total-playback"
-        )
-
-        calculated_summary_context = CalculatedSummaryContext(
+            "20%",
+            2,
+            [],
+        ),
+        (
+            "number-total-playback",
+            "cy",
             "cy",
-            self.schema,
-            self.answer_store,
-            self.list_store,
-            self.progress_store,
-            self.metadata,
-            self.response_metadata,
-        )
-
-        context = calculated_summary_context.build_view_context_for_calculated_summary(
-            current_location
-        )
-
-        self.assertIn("summary", context)
-        self.assert_summary_context(context)
-        self.assertEqual(len(context["summary"]), 6)
-        context_summary = context["summary"]
-        self.assertTrue("title" in context_summary)
-        self.assertEqual(
-            context_summary["title"],
             "We calculate the total of number values entered to be 22. Is this correct?",
-        )
+            "22",
+            2,
+            [],
+        ),
+    ),
+)
+def test_build_view_context_for_currency_calculated_summary(
+    block_id,
+    locale,
+    language,
+    title,
+    value,
+    total_blocks,
+    answers,
+    test_calculated_summary_schema,
+    test_calculated_summary_answers,
+    list_store,
+    progress_store,
+    mocker,
+):
+    mocker.patch(
+        "app.jinja_filters.flask_babel.get_locale",
+        mocker.MagicMock(return_value=locale),
+    )
 
-        self.assertTrue("calculated_question" in context_summary)
-        self.assertEqual(
-            context_summary["calculated_question"]["title"],
-            "Grand total of previous values",
-        )
-        self.assertEqual(
-            context_summary["calculated_question"]["answers"][0]["value"], "22"
-        )
+    current_location = Location(section_id="default-section", block_id=block_id)
+
+    calculated_summary_context = CalculatedSummaryContext(
+        language,
+        test_calculated_summary_schema,
+        test_calculated_summary_answers,
+        list_store,
+        progress_store,
+        metadata={},
+        response_metadata={},
+    )
+
+    context = calculated_summary_context.build_view_context_for_calculated_summary(
+        current_location
+    )
+
+    for answer in answers:
+        test_calculated_summary_answers.add_or_update(answer)
```

Comment:
This doesn't make sense here, this would have to come before initialisation `CalculatedSummaryContext`. We previously had a test when both skipping/not skipping, which we don't cover at the moment. This line at the moment, make no difference to the test.

Reply:
So you mean it needs to move before `CalculatedSummaryContext` or isn't needed at all now?

Reply:
It will need to be before `CalculatedSummaryContext` if it needs to be used. Previous tests covered summary context is as expected when skipping vs not skipping, i.e 1 extra block, which it doesn't look like we cover at the moment.

Reply:
Yeah, the test passes regardless of where the it appears so I guess that single test that adds answers doesn't make a difference?

Reply:
In this instance it doesn't make a difference, I guess I am saying it should?. We don't have explict tests that show skipping vs non skipping which we originally had, this is what is probably missing, as for that, you want answers in the store.